test/unit/gateways/paymill_test.rb

```diff
@@ -181,6 +181,32 @@ def test_successful_store
     assert response.test?
   end
 
+  def test_store_includes_currency
+    expected_currency = 'USD'
+
+    @gateway.expects(:raw_ssl_request).with(
+      :get,
+      store_endpoint(@credit_card, expected_currency, 100),
+      nil,
+      {}
+    ).returns(successful_store_response, successful_purchase_response)
```

Comment:
Since it's been determined that the `store` call no longer works without passing both `currency` and `amount`, what is the reason for creating two separate tests ensuring that these values are passed as opposed to just testing that the `store` calls are successful?